sql/bduni_convert.sql

```diff
@@ -222,19 +226,19 @@ WITH t AS (
     UNION
   SELECT ST_Force2D(ST_EndPoint(geom)) as geom FROM bduni_troncon
 )
-INSERT INTO nodes (lon,lat,geom)
-  SELECT ST_X(t.geom),ST_Y(t.geom),t.geom FROM t WHERE nodes_id(t.geom) IS NULL
+INSERT INTO {output_schema}.nodes (lon,lat,geom)
+  SELECT ST_X(t.geom),ST_Y(t.geom),t.geom FROM t WHERE {output_schema}.nodes_id(t.geom) IS NULL
 ;
 
 -- ############################
 -- REMPLISSAGE DE edges
 -- ############################
 
-INSERT INTO edges
+INSERT INTO {output_schema}.edges
   SELECT
-    nextval('edges_id_seq') AS id,
-    nodes_id(ST_StartPoint(geom)) as source_id,
-    nodes_id(ST_EndPoint(geom)) as target_id,
+    nextval('{output_schema}.edges_id_seq') AS id,
+    {output_schema}nodes_id(ST_StartPoint(geom)) as source_id,
```

Comment:
I've got an error anad I think it's here : missing a dot ^^

Reply:
I just updated the file.

Thanks again for these tests, I'm not able myself to test againt a bduni.

Reply:
u're welcome. ;)